pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -149,5 +151,20 @@ func addMutators(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter k8s_c
 
 	defaultMutator := k8s_webhooks.DefaultingWebhookFor(mgr.GetScheme(), converter)
 	mgr.GetWebhookServer().Register("/default-dubbo-io-v1alpha1-mesh", defaultMutator)
+
+	podEnvInjectMutator := k8s_webhooks.PodMutatingWebhook(
+		func(ctx context.Context, pod *kube_core.Pod) error {
+			RegistryCenterUrl := rt.RegistryCenter().GetURL()
+			if pod.Labels["dubbo.io/name"] == "dubbo-admin" {
```

Comment:
The check condition should be `if pod.Labels["dubbo.io/name"] != ""`?

Reply:
yes, all of the dubbo pod should be injected DUBBO_REGISTRY_ADDRESS env.  I have already modified it.